docs/reference/dqm.rst

```diff
@@ -0,0 +1,41 @@
+.. _dqm:
+
+=========================
+Discrete Quadratic Models
+=========================
+
+:term:`Discrete quadratic model`\ s (DQMs) are described under
+:std:doc:`Discrete Quadratic Models <oceandocs:concepts/dqm>`.
```

Comment:
This should be a short description with the embedded link.

Reply:
I copied this exactly from https://github.com/dwavesystems/dimod/blob/master/docs/reference/bqm.rst

Reply:
Yes, sorry, we agreed that the repos would point to the SDK for concepts.